applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -9,8 +9,11 @@
 #include <zephyr/bluetooth/bluetooth.h>
 #include <zephyr/bluetooth/audio/audio.h>
 #include <zephyr/bluetooth/hci.h>
-#include <../subsys/bluetooth/audio/endpoint.h>
-#include <../subsys/bluetooth/audio/audio_iso.h>
+#include <zephyr/bluetooth/audio/bap.h>
+
+#include <zephyr/bluetooth/audio/bap_lc3_preset.h>
```

Comment:
The newline should be after
#include <zephyr/bluetooth/audio/bap_lc3_preset.h>
and not before?

Reply:
Correct. Done